lona/view.py

```diff
@@ -1,29 +1,90 @@
-from typing import List, Dict
+from typing import (
+    overload,
+    cast,
+    Awaitable,
+    Optional,
+    Iterator,
+    Callable,
+    TypeVar,
+    Union,
+    Tuple,
+    Type,
+    List,
+    Dict,
+    TYPE_CHECKING,
+)
+from concurrent.futures import CancelledError
 import threading
 import asyncio
+import sys
 
-from lona.view_runtime import VIEW_RUNTIME_STATE
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
+from jinja2.nodes import Template
+
+from lona.view_runtime import ViewRuntime, VIEW_RUNTIME_STATE
+from lona.exceptions import ServerStop, UserAbort
+from lona.html.abstract_node import AbstractNode
+from lona.events.input_event import InputEvent
 from lona.shell.shell import embed_shell
+from lona.connection import Connection
+from lona.errors import ClientError
+from lona.request import Request
+
+# avoid import cycles
+if TYPE_CHECKING:
+    from lona.server import LonaServer
+
+
+_T = TypeVar('_T')
+_V = TypeVar('_V', bound='LonaView')
+_HTML = Union[None, AbstractNode, str]
```

Comment:
Also here. Does it make sense to collect such types in a dedicated module to make the reusable?

Reply:
I don't think it's a good idea. None of them truly reusable.
`_T` is just any type.
`_V` is a user view and seems is used only in this file.
`_HTML` is also used only here. Different type `Union[AbstractNode, str]` will be used in other files.